tools/nimgrep.nim

```diff
@@ -50,44 +58,135 @@ Options:
                -b:N   print N lines of leading context before every match
   --context:N, -c:N   print N lines of leading context before every match and
                       N lines of trailing context after it
+  --sortTime          order files by the last modification time -
+       -s[:asc|desc]  - ascending (default: recent files go last) or descending
   --group, -g         group matches by file
   --newLine, -l       display every matching line starting from a new line
+  --cols[:N]          limit max width of lines from files by N characters (off)
```

Comment:
can be done in future work but `ripgrep` can also show column (of match), not just line; it makes it easier when jump to line+col in an editor or on terminal

Reply:
added to TODO